src/index.ts

```diff
@@ -48,14 +50,143 @@ export default class _CONNECTOR_NAME_Connector extends BaseConnector<
     return event
   }
 
-  // Your actions
-  action1(bar: string): void {
-    // Your implementation here
+  async handle(req: Request, res: Response): Promise<boolean> {
+    const eventType = req.headers['x-projects-event']
+    const webhookPayload = req.body
+
+    // TODO validate the payload with the signature hash
+    const eventsToExecute = Object.values(this.eventConfigurations).filter(
+      (e) => e.options.type === eventType,
+    )
+    for (const event of eventsToExecute) {
+      await this.app.handleEvent(event.id, webhookPayload)
+    }
+
+    if (eventsToExecute.length == 0) {
+      res.send({ text: 'Error' })
+    }
+    res.sendStatus(200)
```

Comment:
If `eventsToExecute` is empty, we are calling `res.send` and then `res.sendStatus` - which isn't great